server/src/main/java/io/druid/server/coordinator/helper/DruidCoordinatorCleanupOvershadowed.java

```diff
@@ -82,6 +83,19 @@ public DruidCoordinatorRuntimeParams run(DruidCoordinatorRuntimeParams params)
             stats.addToGlobalStat("overShadowedCount", 1);
           }
         }
+
+        for (LoadQueuePeon loadQueue : coordinator.getLoadManagementPeons().values()) {
```

Comment:
@drcrallen you can not, you want to make sure that what is overshadowing is loaded.